### ISSUE TITLE
Wifi connection stopped after backup started

### ISSUE THREAD

**Issue by jackoGT** [outdoorbits/little-backup-box#235]
Hi,
when i start the backup from usb to usb, the wifi connection drops and the hotspot stops.
I'm using a Raspberry Pi 4 (2Gb) with the latest version of Raspi Os.

**Comment by outdoorbits** [comment]
Sounds like a problem of power supply. Did you try to use a powered usb switch?

**Comment by jackoGT** [comment]
No. I have tried powering the Raspberry Pi 4 both with a powerbank that provides 4.5A amperage with a 5v voltage and with a 5V and 3.4A mains power supply.

**Comment by outdoorbits** [comment]
The RPI has very poor power supply to USB devices. In discussions there is one tweak soldering an extra cable. I didn't try but at least if I use more than one USB device I use my powered hub.

**Comment by jackoGT** [comment]
But I didn't say to use a hub to backup...
I recently installed Lbb on my new Raspberry Pi 4 2Gb, already from the first backup, after inserting the destination and source memory (so when the backup actually starts), the hotspot's wifi connection collapses.
The Raspberry is powered by a 5v 3A mains adapter or a power bank that provides 5v 4.5A (maximum).
Maybe this problem is due to an error in the compilation of some component being updated?

**Comment by outdoorbits** [comment]
WiFi is completely managed by Comitup. As long as you don't connect Comitup to another hotspot, it will serve as a hotspot. There are no interactions between backup and providing the hotspot.
If you need both, the hotspot of Comitup and connect to a WiFi at the same time, your need a second WiFi adapter.
But if I got you right, Comitup doesn't connect to a WiFi and the hotspot gets lost after connecting two USB drives? In this case I would expect, it's a problem of internal power distribution.

**Comment by jackoGT** [comment]
I have explained myself wrongly.
When I connect the phone via wifi to manage the lbb backup, it drops after starting the backup.
The backup runs correctly but I can't manage the raspberry through the wifi connection.
Only after restarting the raspberry does the wifi connection get restored (but as soon as I start a new backup this falls out).

**Comment by outdoorbits** [comment]
Actually I can't explain what's happening. But whenever USB devices are in use and strange things happen - usually a USB device gets lost - my reflex is to check power problems as this is a problematic point.
The only possible software interaction I could imagine is if there was a bug in VPN handling. But you are doing a local backup, right? And did you configure VPN at all?

**Comment by outdoorbits** [comment]
Do you also lose WiFi if you backup to internal storage?

**Comment by jackoGT** [comment]
No, I do not use any vpn.
I tried backing up from the usb to the internal memory and the problem I reported disappears.
If I try to make a new backup from the usb memory to another usb memory, the problem reappears.
However, I have discovered that if the backup from usb to usb is left to work peacefully (even without any display of what has actually been done as the wifi connection has dropped), after a while the wifi connection reappears.
